src/components/Input/styles/BackdropV2.css.ts

```diff
@@ -152,10 +151,7 @@ export const FocusUI = styled('div')`
     top: -1px;
   }
 
-  &.is-error {
-    box-shadow: 0 0 0 ${config.focusErrorOutlineWidth}px
-      ${config.focusErrorOutlineColor};
-  }
+  ${makeFocusStateStyles};
```

Comment:
just curious, emotion will execute the function even though there is no direct evocation here? )